Makefile

```diff
@@ -0,0 +1,24 @@
+SHELL=/bin/bash -euo pipefail
```

Comment:
Actually this seems to be needed. I get a syntax error when I remove it - I suspect this is because it runs the install script here:

```make
.bin/ory: Makefile
		bash <(curl https://raw.githubusercontent.com/ory/meta/master/install.sh) -d -b .bin ory v0.1.33
		touch .bin/ory
```

Reply:
Are you sure? I don't get errors when I remove the `SHELL` line or the `export PATH` line, or both. Can you please post the syntax error you get?

Reply:
```
bash <(curl https://raw.githubusercontent.com/ory/meta/master/install.sh) -d -b .bin ory v0.1.33
/bin/sh: -c: line 0: syntax error near unexpected token `('
/bin/sh: -c: line 0: `bash <(curl https://raw.githubusercontent.com/ory/meta/master/install.sh) -d -b .bin ory v0.1.33'
make: *** [.bin/ory] Error 2
```

Reply:
It also pops up in the github actions run:
https://github.com/ory/examples/runs/7803148964?check_suite_focus=true#step:8:35

Reply:
Thanks for the pointer! Yeah, the Makefile uses an unnecessary bashism here. This can be simplified to work with all shells:

```
curl https://raw.githubusercontent.com/ory/meta/master/install.sh | bash -s -- -d -b .bin ory v0.1.33
```

Then you don't need to set the shell in the Makefile. But I didn't know that one can override the shell in Makefiles. That's going to come in handy in the future!

Reply:
The old version runs `curl https://raw.githubusercontent.com/ory/meta/master/install.sh` in a subshell (that's what the paretheses around it do). The curl command downloads `install.sh` and prints its content. The `<` operator pipes the output of the subshell into the stdin stream of the `bash` executable and then runs that executable (in the same shell, not a subshell).

My version first runs curl to download and print the content of `install.sh` to stdout. I pipe that output into the stdin of the `bash` executable using the `|` operator. That executable takes the commands to run from stdin (since we didn't give it the name of a file containing commands to execute) and executes them. No subshells are involved.

Both versions use UNIX primitives but the old version ends up being so complicated that the standard shell `/bin/sh` cannot parse it anymore and gives up. Bash is a more modern and sophisticated shell and can untangle the two nested shell calls correctly. The advantage of my version is that it simplifies the way both binaries are called and connected so much that the standard shell can understand it. This also makes it easier for humans to grok.